flow/designs/asap7/ibex/config.mk

```diff
@@ -1,43 +1,29 @@
 $(info [INFO-FLOW] IBEX Design)
 
-DESIGN_DIR                   := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
-DESIGN_PDK_HOME              := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+export PLATFORM               = asap7
+export CORNER                ?= BC
```

Comment:
does asap7 have corners?

Reply:
how was this resolved?

Reply:
please check platforms/asap7/config.mk for CORNER library selection

Reply:
The default corner is already defined in platforms/asap7/config.mk so why are you repeating it in each design?  This line should be removed.

Reply:
@maliberty Example design should keep this variable will help user to select required CORNER. If we remove user will use Fast corner library only as per platform config.mk. Is that ok?